<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>317</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>536</width>
    <height>317</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>536</width>
    <height>317</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Midi Synth</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="deviceBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>521</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Devices</string>
    </property>
    <widget class="QComboBox" name="midiCombo">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>14</y>
       <width>401</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Midi device</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="osc_A">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>521</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Oscillators</string>
    </property>
    <widget class="QComboBox" name="osc1_wave">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Osc_A</string>
     </property>
    </widget>
    <widget class="QComboBox" name="osc2_wave">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Osc_B</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>321</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QDial" name="level_OscA">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="sliderPosition">
         <number>50</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDial" name="level_OscB">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>50</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QDial" name="level_Noise">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>20</number>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QDial" name="level_Detune">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>50</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Osc_A Lev</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Osc_B Lev</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Noise Lev</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label_9">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Osc B detune</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>101</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Low Pass</string>
    </property>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>CutOff freq</string>
     </property>
    </widget>
    <widget class="QDial" name="level_Cutoff">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>61</width>
       <height>71</height>
      </rect>
     </property>
     <property name="minimum">
      <number>10</number>
     </property>
     <property name="maximum">
      <number>1500</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_value_Cutoff">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>[Hz]xxx</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>411</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>ADSR</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>411</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QDial" name="atk_time">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>5000</number>
        </property>
        <property name="value">
         <number>50</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDial" name="dcy_time">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>2000</number>
        </property>
        <property name="value">
         <number>1000</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QDial" name="dcy_lev">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="value">
         <number>1000</number>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QDial" name="sus_dur">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>5000</number>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QDial" name="sus_lev">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="value">
         <number>1000</number>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QDial" name="rls_time">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>5000</number>
        </property>
        <property name="value">
         <number>500</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Atk time</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_10">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Dcy time</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_11">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Dcy lev</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label_13">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Sus dur</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="label_14">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Sus lev</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QLabel" name="label_12">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Rls time</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>536</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
